docs/core/entity.md

```diff
@@ -104,6 +104,10 @@ The following properties are used and controlled by Home Assistant, and should n
 
 ## Entity naming
 
+:::tip
+Don't set an entity's name to a hard coded English string, instead, the name should be [translated](/docs/internationalization/core#name-of-entities).
```

Comment:
The point is that integration authors should not add hard coded natural language strings to their integration. Is that clear, or should this be rephrased?

Reply:
Good comments, I'll update the examples 👍